src/components/auth/Login.js

```diff
@@ -45,18 +45,38 @@ const Login = () => {
                 <LoginContainer>
                     <section className="loginform">
                         <form onSubmit={ev => handleSubmit(ev, login)}>
-                            <fieldset>
-                                <legend>Login</legend>
-                                <label htmlFor="inputUsername" className="sr-only">Username</label>
-                                <input type="text" id="inputUsername" onChange={handleUsername}
+                            <FormControl as="fieldset">
+                                <Heading as="legend" size="lg" mb="3" > Login</Heading>
+                                <Input
+                                    autoFocus
+                                    isRequired
+                                    type={"text"}
```

Comment:
When there is nothing special/dynamic about an attribute better to use the standard form:
```suggestion
                                    type="text"
```

Reply:
done!